test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
+      var options = {
+        krabby: {
+          tests: [],
+          reports: [],
+          'extra-option': true
+        }
+      };
+
+      var parsedOptions = krabby._getOptions(options);
+      assert(_.isEqual(options.krabby, parsedOptions));
+    });
+    it('throws an Error it can\'t find a config', function() {
+
+      var krabby = proxyquire('../..', {
+        fs: {
+          'existsSync': function() {
+            return;
+          }
+        },
+        'find-parent-dir': {
+          sync: function() {
+            return;
+          }
+        }
+      });
+
+      assert.throws(function() {
+        krabby._getOptions({});
+      }, /can't find the root of the project\. go find a package\.json file/);
```

Comment:
Maybe we should update this Error message. I don't think I'd know what to do with it if I were just starting out.